From: Jan Engelhardt <ej@inai.de>
Date: 2024-09-22 10:07:26.477018956 +0200

Reduce the autoconf requirement by reverting modern macro use a bit.

diff --git configure.ac configure.ac
index ab78271..de17dae 100644
--- a/configure.ac
+++ a/configure.ac
@@ -1,11 +1,11 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 
 AC_INIT([libacm],[1.3],[markokr@gmail.com])
 AC_CONFIG_SRCDIR([src/libacm.h])
 AM_INIT_AUTOMAKE
 AC_CONFIG_HEADERS([config.h])
-AC_CONFIG_MACRO_DIRS([m4])
+AC_CONFIG_MACRO_DIR([m4])
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
@@ -45,7 +45,7 @@ AC_C_CONST
 AC_TYPE_SIZE_T
 
 dnl Checks for library functions.
-AC_CHECK_INCLUDES_DEFAULT
+AC_HEADER_STDC
 
 dnl Plugin configuration
 PKG_PROG_PKG_CONFIG
